index.html

```diff
@@ -1098,7 +1098,7 @@
 * +-------------------------------------------------+ *
 *******************************************************
 
-The intensity of the EDF is controlled by a luminance and a color multiplier. The **`emission_luminance`** parameter controls the luminance the emissive layer would have when **`emission_color`** is set to (1, 1, 1) and in the absence of coat and fuzz. The **`emission_color`** acts as a multiplier, thus the resulting luminance may be less than the input parameter, or even zero if the color multiplier is set to (0, 0, 0).
+The intensity of the EDF is controlled by a luminance and a color multiplier. The **`emission_luminance`** parameter controls the luminance the emissive layer would have when **`emission_color`** is set to (1, 1, 1) and in the absence of coat and fuzz. The **`emission_color`** acts as a multiplier, i.e. the HDR emission in the model color space is defined to have a color given by **`emission_color`** * **`emission_luminance`**, thus the resulting luminance may be less than the input parameter, or even zero if the **`emission_color`** is set to (0, 0, 0).
```

Comment:
Minor remark: I think it would be clearer to use `\times`, or at least have the whole expression marked as using a monospace font.

Reply:
Though we do it this way (with a roman asterisk) in a couple of other places, e.g. 

**`subsurface_radius`** * **`subsurface_radius_scale`**
**`base_weight`** * **`base_color`**

I personally don't mind (I think it looks a bit worse if all monospace though).

Reply:
Ah well, let's prefer consistency then.